contracts/examples/digital-cash/src/deposit_info.rs

```diff
@@ -0,0 +1,13 @@
+use elrond_wasm::{api::BigUintApi, types::TokenIdentifier};
+use elrond_wasm::types::Address;
+
+elrond_wasm::derive_imports!();
+
+#[derive(NestedEncode, NestedDecode, TopEncode, TopDecode, TypeAbi)]
+pub struct DepositInfo<BigUint: BigUintApi>
+{
+	pub amount : BigUint,
+	pub depositor_address : Address,
+	pub expiration: u64,
```

Comment:
Usually, when I see a field with u64 and it implies time, I immediately assume unix timestamp. But this seems to be measured in rounds instead, so rename to expiration_round or something similar.

Reply:
Done!